src/Semoda/Semoda/ViewModels/DashboardPageViewModel.cs

```diff
@@ -13,6 +18,25 @@ public class DashboardPageViewModel : ViewModelBase
         /// </summary>
         public DashboardPageViewModel() : base(true)
         {
+            IPerformanceDataService dataService = ServiceProvider.GetRequiredService<IPerformanceDataService>();
```

Comment:
This is just a show off of the functionality. Will be removed, when there is a view to represent the data